### ISSUE TITLE
[Discussion] about v 1.x

### ISSUE THREAD

**Issue by remicollet** [jedisct1/libsodium-php#154]
Hi,

Latest 1.0.6 is the only version compatible with PHP 5.x (5.6 is still maintained until Dec 2018, and still widely used).

I understand you only want to maintain v2.x, but, I think it can make sense to maintain 1.x only for critical (security) issues.

So would be nice to have a v1.x branch

Needed in this branch: ecbf0f15206d6898b26fc4c12a5c03a2e1609995 and e529d49b4ab0a875333eff867722207de97bcb76

Notice, as extension name have changed, and as there is absolutely no symbol conflict, it is possible to load both version with PHP 7.x, which can have some value (at least to test some compat library)

```
$ php -v
PHP 7.2.0RC3 (cli) (built: Sep 28 2017 11:07:41) ( NTS )
...
$ php -m | grep sod
libsodium
sodium
```

**Comment by jedisct1** [comment]
I just released a 1.0.7 version.

The Travis tests seem to be passing, but that's all I know.

**Comment by remicollet** [comment]
thanks
